db/migrations/8-trips-auto-approve.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE trips
+ADD COLUMN auto_approved_members INTEGER NOT NULL DEFAULT 0;
+
+ALTER TABLE trip_members
+ADD COLUMN added_at INTEGER NOT NULL DEFAULT (strftime('%s', 'now'));
```

Comment:
Don't think we need this—the order they're in the table based on the `ROWID` is going to be the order that they were added at. To create new `ROWID`s, SQLite basically does `SELECT max(id) + 1`.

`trip_members` does not have an explicit `INTEGER PRIMARY KEY`, but it still has a hidden one, essentially, because `INTEGER PRIMARY KEY` is just an alias for [ROWID](https://www.sqlite.org/lang_createtable.html#rowid), which SQLite will always include unless you explicitly ask it not to.

Reply:
In this case `trip_members` table has a `PRIMARY KEY (trip, user)` and so it ends up sorting by the trip ID then user ID, instead of order added. It definitely would've been nice to just rely on insertion order.

Reply:
SQLite works a little differently in this regard (namely, you always have ROWID). See: https://www.sqlite.org/rowidtable.html